packages/renderer/src/lib/preferences/PreferencesRenderingItemFormat.svelte

```diff
@@ -14,25 +14,42 @@ import type { IConfigurationPropertyRecordedSchema } from '../../../../main/src/
 import Markdown from '../markdown/Markdown.svelte';
 import { getInitialValue, getNormalizedDefaultNumberValue } from './Util';
 
-let invalidText: string | undefined = undefined;
-export let invalidRecord = (_error: string) => {};
-export let validRecord = () => {};
-export let updateResetButtonVisibility = (_recordValue: any) => {};
-export let resetToDefault = false;
-export let enableAutoSave = false;
-
-export let setRecordValue = (_id: string, _value: string | boolean | number) => {};
-export let enableSlider = false;
-export let record: IConfigurationPropertyRecordedSchema;
-export let initialValue: Promise<any>;
-export let givenValue: unknown = undefined;
+let invalidText: string | undefined = $state(undefined);
```

Comment:
suggestion (non-blocking): state should be defined after props ?

Reply:
instruction moved